src/index.ts

```diff
@@ -10,3 +10,4 @@ export * from './preview/PdfPreview';
 /** Utils */
 export * from './preview/PreviewManager';
 export * from './preview/PreviewWrapper';
+export * from './utils/utils';
```

Comment:
We are exporting the util, so it is available form outside. Do we need to keep it internal or we leave it available for external usage?